tools/actions/live-common-affected/build/main.js

```diff
@@ -18895,13 +18895,13 @@ function main(ref) {
       const list = [
         ...new Set(
           stdout.split("\n").map((line) => {
-            const m = line.match(/live-common\/src\/([^/]+)\/([^/]+)/);
+            const m = line.match(/(live-common\/src\/([^/]+)\/([^/]+))|(coin-modules\/([^/]+))/);
             if (m) {
-              const [, first, second] = m;
+              const [zero, first, second] = m;
```

Comment:
Mixing ordinals and cardinals seems like a weird naming. What about:
```suggestion
              const [first, second, third] = m;
```